packages/cashier/src/stores/account-transfer-store.ts

```diff
@@ -473,9 +466,11 @@ export default class AccountTransferStore {
 
                 const { account_id, login } = this.root_store.traders_hub?.selected_account;
 
+                const is_traders_hub = window.location.pathname === routes.traders_hub;
```

Comment:
why changed the condition ?
`const is_from_outside_cashier = !location.pathname.startsWith(routes.cashier);`

Reply:
As i discussed with @heorhi-deriv, we plan to use traders hub path. But this can work as well by not changing much. Will use this